### PR TITLE
New node selector labels

### PR DESCRIPTION
**Pull request by DTadrzak**
New node selector label layout and few changes in service dependencies.

### CODE REVIEW

File: stackanetes-services/memcached/memcached.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 type: deployment
 name: memcached
+label: control-plane
 image: memcached
 command: "su  memcache -c 'memcached'"
```
Comment (PiotrProkop) [inline stackanetes-services/memcached/memcached.yml]: change to memcached

File: stackanetes-services/compute/compute-node.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 type: daemonset
 name: compute-node
+label: compute-node
```
Comment (Quentin-M) [inline stackanetes-services/compute/compute-node.yml]: Should we prefix them with `openstack-` to avoid label collision on big enterprise clusters?

Comment (DTadrzak) [inline stackanetes-services/compute/compute-node.yml]: Agree

File: stackanetes-services/memcached/memcached.yml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 type: deployment
 name: memcached
+label: openstack-control-plane
 image: memcached
-command: "su  memcache -c 'memcached'"
+command: "memcached"
```
Comment (pawelsocha) [inline stackanetes-services/memcached/memcached.yml]: So, docker image drop root privileges to user memcache?

Comment (DTadrzak) [inline stackanetes-services/memcached/memcached.yml]: Ofc

**Pull request merged**